src/main/java/edu/cornell/kfs/sys/batch/service/impl/CuAutoDisapproveDocumentsServiceImpl.java

```diff
@@ -47,14 +49,14 @@
     private static final String TAB = "\t";
 
     private RouteHeaderService routeHeaderService;
+    private PersonService personService;
+    
 
-    protected boolean processAutoDisapproveDocuments(String principalId, String annotation, Date documentCompareDate) {
+    protected boolean processAutoDisapproveDocuments(String principalId, String annotation) {

```

Comment:
Since this is overriding a superclass method, I would suggest adding the "Override" annotation for clarity.